h5c/vic/src/vic-webapp/src/app/vch-view/vch-view.template.html

```diff
@@ -67,7 +67,7 @@
 
   <clr-dg-row *ngFor="let item of vchs">
     <clr-dg-action-overflow class="{{item.name}}">
-      <button class="action-item" (click)="deleteVch(item)" class="{{item.name}}">Delete</button>
+      <button class="action-item {{item.name}}" (click)="deleteVch(item)">Delete</button>
```

Comment:
I'm not quite clear why we are using the vch name as a class in the first place?

Reply:
I think you or jkim added it for e2e tests purposes.

Reply:
https://github.com/vmware/vic-ui/blob/master/h5c/vic/src/vic-webapp/e2e/app.po.ts#L139

Reply:
Oh right. Ha. thanks.